ImageKitSample/Program.cs

```diff
@@ -0,0 +1,426 @@
+
+            // Bulk Delete
+            List<string> ob3 = new List<string>();
+            ob3.Add("fileId_1");
+            ob3.Add("fileId_2");
+            ResultFileDelete resultFileDelete = imagekit.BulkDeleteFiles(ob3);
+
+            // Copy File
+            CopyFileRequest cpyRequest = new CopyFileRequest
+            {
+                SourceFilePath = "path_1",
+                DestinationPath = "path_2",
+                IncludeFileVersions = true,
+            };
+            ResultNoContent resultNoContent = imagekit.CopyFile(cpyRequest);
+
+            // MoveFile
+            MoveFileRequest moveFile = new MoveFileRequest
+            {
+                SourceFilePath = "path_1",
+                DestinationPath = "path_2",
+            };
+            ResultNoContent resultNoContentMoveFile = imagekit.MoveFile(moveFile);
+
+            // RenameFile
+            RenameFileRequest renameFileRequest = new RenameFileRequest
+            {
+                FilePath = "path_1",
+                NewFileName = "file_name",
+                PurgeCache = false,
+            };
+            ResultRenameFile resultRenameFile = imagekit.RenameFile(renameFileRequest);
+
+            #endregion
+
+            #region  Tags
+            TagsRequest tagsRequest = new TagsRequest
+            {
+                Tags = new List<string>
+                {
+                    "tag_1",
+                    "tag_2",
+                },
+                FileIds = new List<string>
+                {
+                    "field_1",
+                },
+            };
+            ResultTags resultTags = imagekit.AddTags(tagsRequest);
+
+            TagsRequest removeTagsRequest = new TagsRequest
+            {
+                Tags = new List<string>
+                {
+                    "tag_1",
+                    "tag_2",
+                },
+                FileIds = new List<string>
+                {
+                    "field_1",
+                },
+            };
+            ResultTags removeTags = imagekit.RemoveTags(removeTagsRequest);
+
+            AiTagsRequest removeAiTagsRequest = new AiTagsRequest
+            {
+                AiTags = new List<string>
+                {
+                    "tag_1",
+                    "tag_2",
+                },
+                FileIds = new List<string>
+                {
+                    "field_1",
+                },
+            };
+            ResultTags removeAiTags = imagekit.RemoveAiTags(removeAiTagsRequest);
+
+            #endregion
+
+            #region FileVersionRequest
+            DeleteFileVersionRequest delRequest = new DeleteFileVersionRequest
+            {
+                FileId = "file_Id",
+                VersionId = "version_Id",
+            };
+            ResultNoContent resultNoContent1 = imagekit.DeleteFileVersion(delRequest);
+
+            // RestoreFileVersion
+            Result result = imagekit.RestoreFileVersion("file_Id", "version_Id");
```

Comment:
`Result result = imagekit.RestoreFileVersion("file_Id", "version_Id");` this convention is different from the one used in delete file version requeest i.e.
```
DeleteFileVersionRequest delRequest = new DeleteFileVersionRequest
            {
                FileId = "file_Id",
                VersionId = "version_Id",
            };
```

 We should stick to one convention in the sample file.

Reply:
Still pending.